docs/guides/high-availability/examples/generated/ispn-site-b.yaml

```diff
@@ -415,14 +419,17 @@ metadata:
     infinispan.org/monitoring: 'true' # <2>
 spec:
   replicas: 3
+  jmx:
+    enabled: true
 # end::infinispan-single[]
 # end::infinispan-crossdc[]
   # This exposes the http endpoint to interact with its caches - more info - https://infinispan.org/docs/stable/titles/rest/rest.html
   # We can optionally set the host in the below expose yaml block, otherwise it will be set to a default naming pattern.
   expose:
     type: Route
   configMapName: "cluster-config"
-  image: quay.io/infinispan/server:15.0.7.Final
+  image: quay.io/infinispan-test/server:15.0.x
+  version: 15.0.4
```

Comment:
The image and version field changes need to be reverted.

Reply:
It seems this part of the document is never used in HA guide and this is what we have in our helm chart by default. I would vote for keeping this as is, at least for this PR.